include/eve/memory/pointer.hpp

```diff
@@ -8,11 +8,71 @@
 #pragma once
 
 #include <eve/memory/aligned_ptr.hpp>
+#include <eve/detail/kumi.hpp>
 
 #include <concepts>
 
+namespace eve
+{
+  template<typename T, typename Enable=void>
+  struct pointer_traits
+  {
+    using value_type = void;
```

Comment:
Why void?
I mean, let's just not do anything.

And then just require `pointer_traits::value_type`